st2common/st2common/util/virtualenvs.py

```diff
@@ -166,8 +166,12 @@ def install_requirements(virtualenv_path, requirements_file_path):
     exit_code, stdout, stderr = run_command(cmd=cmd, env=env)
 
     if exit_code != 0:
-        raise Exception('Failed to install requirements from "%s": %s (stderr: %s)' %
-                        (requirements_file_path, stdout, stderr))
+        def safetystring(s):
```

Comment:
One option would perhaps also be to use `st2common.util.compact.to_unicode` function.

That's also a good reason for a test case - if we have a test case we can also easily see if existing `to_unicode` utility function works or if we need this new one :)